spec/controllers/applications/students_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe Applications::StudentsController, type: :controller do
```

Comment:
can you add a `render_views` to it? That way we have a regression test for errors in the templates.